float64[] pose_x
float64[] pose_y
float64[] pose_w

uint8 map_control
uint8 navigation_task
uint8 hardware_control
string map_id
